hangman.rb

```diff
@@ -0,0 +1,69 @@
+# (2) Write a program hangman.rb that contains a function called hangman. 
+# The function's parameters are a word and an array of letters. 
+# It returns a string showing the letters that have been guessed. 
+# Call the function from within your program so that you know that it works.
+
+# Example: hangman("bob",["b"]) should evaluate to "b_b"
+# Example: hangman("alphabet",["a","h"]) should return "a__ha___"
+
+class Game
+
+  attr_reader :word
+
+    def initialize(word)
+        @word = word.downcase
+        @display_word = "_" * word.length
+       
+    end
+
+   
+    def guess_the_letter(g_letter)
+        g_letter.downcase
+        @word.split("").each_with_index do |w_letter, index|
+            if g_letter == w_letter
+                @display_word[index] = g_letter
+                puts "Your progress so far" 
+        p @display_word
+            end
+          if !@word.include? (g_letter)
+            puts "Please try again"
+          end
+        end
+    end
+
+  
+    def game_won?
+      if    @word == @display_word
+            puts "Congratulations Player 2, you won!"
+            
+          
+      elsif
```

Comment:
without a condition, we don't have to use `elsif`. It is enough with just `else`